Gemfile.lock

```diff
@@ -10,22 +10,22 @@ GEM
   remote: https://rubygems.org/
   specs:
     eventmachine (1.2.7)
-    faye-websocket (0.10.7)
+    faye-websocket (0.10.9)
       eventmachine (>= 0.12.0)
       websocket-driver (>= 0.5.1)
     mustermann (1.0.3)
-    rack (2.0.7)
-    rack-protection (2.0.5)
+    rack (2.0.8)
+    rack-protection (2.0.7)
       rack
-    sinatra (2.0.5)
+    sinatra (2.0.7)
       mustermann (~> 1.0)
       rack (~> 2.0)
-      rack-protection (= 2.0.5)
+      rack-protection (= 2.0.7)
       tilt (~> 2.0)
-    tilt (2.0.9)
-    websocket-driver (0.7.0)
+    tilt (2.0.10)
```

Comment:
### [tilt](http://github.com/rtomayko/tilt/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.9 → 2.0.10

[_~~change-log~~, [source-code](http://github.com/rtomayko/tilt)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/rtomayko/tilt/compare/v2.0.9...v2.0.10).

These are the individual commits:
- (857deb0) [Remove test files from bundled gem (#339)](https://github.com/rtomayko/tilt/commit/857deb0641295c7dd1324330027752497c012c0c)
- (f1c7293) [Test on Ruby 2.5 (#341)](https://github.com/rtomayko/tilt/commit/f1c7293d629473720a1267088774330d88d38b01)
- (a7e0018) [readme: add syntax highlight for code blocks (#342)](https://github.com/rtomayko/tilt/commit/a7e0018678ab5151f47b662628ced6388e56d0b4)
- (dbb4df9) [Fix warning when using yield in templates on ruby 2.7](https://github.com/rtomayko/tilt/commit/dbb4df94e5bc6e533fc2ed09b2fd70df39c049c3)
- (be625c8) [If scope is a Module/Class, treat as scope_class](https://github.com/rtomayko/tilt/commit/be625c89a9e3717fe23f90f9ded0c8a503b66a7d)
- (b0b84e4) [Add test for rending using a class as scope](https://github.com/rtomayko/tilt/commit/b0b84e4ce7d963d27cda5a908d49eaa4204fda3f)
- (84f3a74) [Update CHANGELOG](https://github.com/rtomayko/tilt/commit/84f3a7425909b132f5cf2d2e3ee72595263272ea)
- (6d15120) [v2.0.10](https://github.com/rtomayko/tilt/commit/6d1512061ecf88c1a63a1821a121a2a7c3f37326)


</details>